### PR TITLE
Move "recovery" to "active" on reset

### PR DESCRIPTION
**Pull request by jimmykarily**
Fixes https://github.com/kairos-io/kairos/issues/2051

### CODE REVIEW

**Comment by jimmykarily** [comment]
> ```
> [  114.886782] EXT4-fs (dm-0): unmounting filesystem 9b2b734b-0244-4ed2-8f3e-ef1e94e07035.
> INFO[2024-02-20T15:54:18Z] Formatting 'COS_PERSISTENT' partition        
> ERRO[2024-02-20T15:54:20Z] failed to reset uki: 1 error occurred:
> 	* copying recovery to active: deleting role active: remove /efi/EFI/kairos/active.efi: read-only file system
>  
> 1 error occurred:
> 	* copying recovery to active: deleting role active: remove /efi/EFI/kairos/active.efi: read-only file system
> ```
> 
> something needs to be mounted rw before the reset (?)

fixed

File: pkg/uki/reset.go [hunk 1]
```diff
@@ -55,6 +58,19 @@ func (r *ResetAction) Run() (err error) {
 		}
 	}
 
+	// REMOUNT /efi as RW (its RO by default)
+	umount, err := e.MountRWPartition(r.spec.Partitions.EFI)
```
Comment (mauromorales) [inline pkg/uki/reset.go:62]: maybe just change the name of the var to `mount` instead of `umount`

Comment (jimmykarily) [reply pkg/uki/reset.go:62]: That's really the *unmount* function. It's returned by the `MountRWPartition` : https://github.com/kairos-io/kairos-agent/blob/8c353a95120ab6bc1baf8344164a3905c2bfd18d/pkg/elemental/elemental.go#L167

Comment (mauromorales) [reply pkg/uki/reset.go:62]: gotcha, thanks for clarifying

**Review by mauromorales — APPROVED** [review]

**Pull request merged**